v1.16/QKE/README.md

```diff
@@ -0,0 +1,42 @@
+## How to Reproduce
+
+### Login 
+
+Login to [QingCloud Console Platform](https://console.qingcloud.com/) Website with your own qingcloud account.
```

Comment:
`Sign in to [QingCloud Console]() with you own account.`

Reply:
ok